README.rst

```diff
@@ -58,12 +58,164 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+
+In order to use the **nidaqmx** package, you must have at least one DAQ device installed on your
+machine. X series DAQ device (eg: PCIe-6363 or USB-6351) is recommended, physical or
+simulated devices are supported. You can use the **NI MAX** or **NI Hardware Configuration Utility**
+to verify and configure your devices.
+
+Verify and configure devices on **NI MAX**:
+
+.. image:: max_device_name.png
+  :alt: NI MAX Device Name
+  :align: center
+  :width: 800px
+
+Verify and configure devices on **NI Hardware Configuration Utility**:
+
+.. image:: hwcu_device_name.png
+  :alt: NI HWCU Device Name
+  :align: center
+  :width: 800px
+
+Virtual channels and tasks are fundamental components of NI-DAQmx.
```

Comment:
This paragraph has nothing to do with the section it's in. Is it supposed to introduce the next two sections? If so, the document structure should reflect that, like have a section for virtual channels and tasks that introduces them, then sub-sections for each one. 

Also, this paragraph introduces virtual channels, then tasks, but the following sections are in the reverse order.

Reply:
It was an introduction to Tasks and Channels sections. I have removed this paragraph as I find it's not needed.